core/shortcut_urls.py

```diff
@@ -0,0 +1,10 @@
+from django.urls import path, re_path
+
+from core.views import UrlRedirectView
+
+
+urlpatterns = [
+
+    re_path(r'^(?P<url_shortener>.*)/$', UrlRedirectView.as_view()),
```

Comment:
Do you need a terminating slash to trigger the redirect? Preferably it'd work with both

Reply:
Squire (/Django) always automatically adds a / if one is not present in the requested url, so it's best to assume it will always end with a /

Reply:
If you are referring to the $ at the end, it's best to not catch any extra material after the url, and this way we can be sure that is not the case.